scripts/is/intg/run-int-test.sh

```
@@ -0,0 +1,289 @@
+rm -rf $PRODUCT_REPOSITORY_PACK_DIR
+mkdir -p $PRODUCT_REPOSITORY_PACK_DIR
+log_info "Copying product pack to Repository"
+ls $TESTGRID_DIR
+rm -rf $TESTGRID_DIR/$PRODUCT_NAME-$PRODUCT_VERSION.zip
+ls $TESTGRID_DIR
+
+# Update deployment.toml in the pack with $env{} placeholders for database configuration
+log_info "Updating deployment.toml with environment variable placeholders"
+wget -q https://raw.githubusercontent.com/Miranlfk/is-test-integration/refs/heads/master/update_db_configs.sh
+cp update_db_configs.sh $TESTGRID_DIR/
+bash $TESTGRID_DIR/update_db_configs.sh $DB_TYPE $PRODUCT_NAME-$PRODUCT_VERSION
```

Shellcheck (0.11.0)

[info] Double quote to prevent globbing and word splitting.

(SC2086)